todo_app/app.py

```diff
@@ -17,6 +18,13 @@ def addtodo():
     title=request.form["title"]
     add_item(title)
     return redirect('/')
+#Application code to mark items as Done
```

Comment:
This kid of spacing makes the code a bit difficult to read. Can you please add a space between `#` and the comment, and a new line between methods/functions

Reply:
Also, in general, it's considered cleaner to _not_ include comments in code. Our goal for writing code should be that it's clear enough to be easily readable and self-explanatory, so that comments become unnecessary.

There are a couple of cases that I can think of where comments _are_ necessary:
* Leaving out TODO comments - to mark that a piece of functionality is not done, but it's not feasible to do it now
* Where a piece of code is counter-intuitive and there's nothing we can do to make it better, so we explain to future developers why we do things a certain way.

I think you can safely remove the comments from your code, though

Reply:
Comments have been removed from app.py and trello_items.py